tests/samplers/test_rejection_sampler.py

```diff
@@ -84,8 +82,8 @@ def test_correct_output_format(which_tokens_accepted: str,
                                     size=(batch_size, 1),
                                     dtype=torch.int64)
 
-    rejection_sampler = RejectionSampler(
-        disable_bonus_tokens=disable_bonus_tokens)
+    rejection_sampler = RejectionSampler(disable_bonus_tokens=False,
```

Comment:
I feel you can leave this test untouched, and just rename the follow test to "test_flashinfer_backed".